test/utils.js

```diff
@@ -18,6 +18,10 @@ const getRandomString = (length) => {
 
 const getAddresses = (wallets) => wallets.map(({ address }) => address);
 
+const isHardhat = () => {
+    return network.name === 'hardhat';
+};
```

Comment:
```suggestion
const isHardhat = network.name === 'hardhat';
```
doesn't need to be a function

Reply:
updated